runtime/doc/vim_diff.txt

```diff
@@ -675,6 +675,8 @@ Highlight groups:
         \|setlocal winhighlight=StatusLine:StatusLineTerm,StatusLineNC:StatusLineTermNC
         \|else|setlocal winhighlight=|endif
 <
+Normal commands:
```

Comment:
Ditto. Is "normal command" even something that is defined or even mentioned in the documentation?

Reply:
It literally on the first page of `:h`

Reply:
Hmm, you are right. Although I suspect these definitions aren't used that much or consistently in the documentation. "Command-line command" in particular stinks. But if this is the actual definition then I guess that's that.